src/api/routes/controllers/todos.controller.js

```diff
@@ -67,3 +67,14 @@ exports.deleteTodo = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.modifyTodo = async (req, res, next) => {
+  try {
+    const todoId = req.params.id;
+    const title = req.body.title;
```

Comment:
구조분해할당을 사용해서 선언해줄 수도 있겠네요!
```suggestion
    const { id } = req.params;
    const { title } = req.body;
```

Reply:
변수가 하나일 땐, 구조분해 할당을 잘 안 쓰는 편인데, 서지님은 어떻게 작성하세요? 통일시키는게 좋을것 같네요